File: morning snacks/Gcd.java
import java.util.Scanner;
    public class Gcd {
        public static void main(String...Odili) {
            Scanner input = new Scanner(System.in);

    System.out.println("Enter any number ");
        int firstNumber = input.nextInt();

    System.out.println("Enter another Number ");
        int secondNumber = input.nextInt();
            int gcd = 0;         
   
        for(int count = 1; count <= firstNumber && count <= secondNumber; count ++){
            if(firstNumber % count == 0 && secondNumber % count == 0){  
                gcd = count;
                
                

            }
                   
        }
            System.out.print(gcd);
        

        }
    }

File: morning snacks/Gcd.js
numberOne = 12;

numberTwo = 8;

gcd = 0;

for(count = 1; count < numberOne && count <= numberTwo; count ++){
    if(numberOne % count  == 0 && numberTwo % count == 0){
        gcd = count;
    }
   
    

}

console.log(gcd);
